pkg/tuf/signer.go

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tuf
+
+import (
+	"encoding/json"
+)
+
+const (
+	KeyTypeFulcio   = "sigstore-oidc"
+	KeySchemeFulcio = "https://fulcio.sigstore.dev"
```

Comment:
Does anyone recall what the purpose of this is? Is this a problem that this is hardcoded in, such as for staging?

@dlorenc @bobcallaway maybe?

Reply:
It doesn't appear to be referenced anywhere within cosign at least, I'm not sure if it was just exported on accident.

Reply:
Seems to be used with https://github.com/sigstore/cosign/blob/d2d74640720b40caa335346566fcdbc978f57bba/cmd/cosign/cli/policy_init.go#L77

Scheme isn't referenced though. Probably fine to leave.

Reply:
Moved these to unexported in sigstore/sigstore. 👍